### ISSUE TITLE
Unable to import InteractiveBrokersClient after installing nautilus-trader in a virtualenv

### ISSUE THREAD

**Issue by benjaminsingleton** [nautechsystems/nautilus_trader#1310]
# Bug Report

### Expected Behavior

Expected to be able to import `InteractiveBrokersClient` after installing the package (using `poetry add nautilus-trader -E ib`) in a new, clean project. See steps to reproduce below.

### Actual Behavior

```
>>> from nautilus_trader.adapters.interactive_brokers.client.client import InteractiveBrokersClient

FileNotFoundError: [Errno 2] No such file or directory: '/Users/benjamin/Projects/trading-bot/.venv/lib/python3.10/site-packages/pyproject.toml'
```

The problematic code lies here: https://github.com/nautechsystems/nautilus_trader/blob/master/nautilus_trader/adapters/interactive_brokers/client/client.py#L85-L93

```
# Check ibapi package versioning
ibapi_package = "nautilus_ibapi"
ibapi_version_specified = get_package_version_from_toml(PYPROJECT_PATH, ibapi_package, True)
```

It's trying to find a pyproject.toml file that doesn't exist at `PYPROJECT_PATH`. The `PYPROJECT_PATH` variable is set to a directory relative to nautilus-trader's package `__init__.py` file. When nautilus-trader is installed in a virtual environment using poetry, for example, the path won't correctly point to my actual personal project's pyproject.toml.

```
PACKAGE_ROOT = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
PYPROJECT_PATH = os.path.join(PACKAGE_ROOT, "pyproject.toml")
```

Maybe we can instead set an environment variable to point to our personal project's pyproject.toml file? Still, that's a bit annoying because it requires an extra step for the user. It seems these variables are only relevant to the Interactive Broker's adapter. is this package version check really important? Or can there be a different approach?

### Steps to Reproduce the Problem

1. Fresh install of nautilus-trader with IB extras
2. Attempt to import `InteractiveBrokersClient`

```
$ mkdir nautilus-test && \
cd nautilus-test && \
poetry init --no-interaction && \
sed -i '' '/^python = /c\  
python = ">=3.10,<3.12"' pyproject.toml && \
poetry add nautilus-trader -E ib && \
poetry shell

$ python

>>> from nautilus_trader.adapters.interactive_brokers.client.client import InteractiveBrokersClient

FileNotFoundError: [Errno 2] No such file or directory: '/Users/benjamin/Projects/trading-bot/.venv/lib/python3.10/site-packages/pyproject.toml'
```

### Specifications

- OS platform: 13.0
- Python version:  3.10.13
- `nautilus_trader` version: 1.179.0

**Comment by cjdsellers** [comment]
Hi @benjaminsingleton 

Thanks for reporting this!

Unfortunately this is a downstream consequence of IB not providing wheels for their own packages, so we have to repackage and release to PyPI to make it work with Nautilus and poetry https://pypi.org/project/nautilus-ibapi/.

> is this package version check really important?

I agree, that I don't see much value in this check if it can cause these sorts of issues. The intent is to make sure the user has installed the `nautilus_ibapi` package, and not the `ibapi` package.

I'll remove the check for now to unblock this on the next release (probably happening Sunday or Monday). While we reconsider the approach here.

**Comment by benjaminsingleton** [comment]
Sounds good! And yes, very frustrating and odd that IB doesn't provide wheels for their own packages.

**Comment by cjdsellers** [comment]
Pushed this to `develop` in case it was blocking you in any way.

**Comment by cjdsellers** [comment]
Fixed with the latest v1.180.0 release.
